adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -1336,6 +1340,48 @@
                 };
             }
 
+            if (type == 'iframe') {
+                loginIframe.enable = true;
+                iframe = initOptions.iframe;
+                return {
+                    login: function(options) {
+                        var promise = createPromise(false);
+
+                        console.log("login: ", options);
+                        console.log("iframe: ", iframe);
+                        iframe.addEventListener('load', function(event) {
+                            if (iframe.contentDocument) {
+                                var callback = parseCallback(iframe.contentDocument.location.href);
```

Comment:
I'm guessing you are testing this with both Keycloak and the app on localhost? I believe iframe.contentDocument.location is only available if main window and iframe is on the same domain.

Reply:
I am not testing on localhost, but I am testing with the web app and keycloak on the same domain (but a different subdomain).  I had to configure keycloak to set some security headers I don't recall though.